riptable/rt_matplotlib.py

```diff
@@ -0,0 +1,754 @@
+from typing import List, Optional
```

Comment:
For namespace / import hygiene, please add an ``__all__ = []`` here. I know the way you're importing it in the ``riptable/__init__py`` won't pollute the ``rt.`` namespace, but I think it'd be good to have the ``__all__`` in case this module is ever referenced by some other part of riptable in the future. And in a sense, specifying an empty ``__all__`` signals that no other code really _should be_ referencing this module because it's self-contained.